### DISCUSSION TITLE
METEOR images sometimes in gray-scale

### DISCUSSION THREAD

**Discussion by jp112sdl** [jekhokie/raspberry-noaa-v2#456] [Q&A]
Hi everyone!
For a few weeks, the images of METEOR M2 sometimes appear only in a gray-scale.
Both images were taken on the **same day**.

Does this come directly from the satellite or do I have to change a setting in `raspberry-noaa-v2` configuration?

![METEOR-M-2-20211019-100641-122-rectified](https://user-images.githubusercontent.com/18190139/138040507-13a77444-65d0-4a93-96e1-9e85b6666c21.jpg)
![METEOR-M-2-20211019-195441-122-rectified](https://user-images.githubusercontent.com/18190139/138040565-93f50887-16b8-4987-90aa-59d72ae663d5.jpg)

**Comment by dom-robinson** [comment]
Hi @jp112sdl - Im pretty sure it all depends on the 

```
meteor_m2_sun_min_elevation: 6
meteor_m2_sat_min_elevation: 30
``` 

setting. 

In rtl_fm mode - see ```meteor_receiver: 'rtl_fm'``` then if the sun is below the min then it only produces the IR output. If the sun is above then you get the color version. 

in gnuradio mode - see ```meteor_receiver: 'gnuradio'``` then you will typically get all three (natural colour, false color (without IR in the mix) and IR mode all displayed. Note that as of very soon (later today?) that will become the 'default' mode.

**Reply by jp112sdl** [reply]
Sorry for not answering a long time...

I'm using `rtl_fm` mode and yes, as you said

> if the sun is below the min then it only produces the IR output. 

it happens as described.

While I'm just reading your comment, I see in the annotation of my pictures a sun elevation of **-7°**.

That's wrong for that time.
So I compared the sun elevation value with https://www.sonnenverlauf.de/ for my location.
It's completely different.

i.e.: while a received image says **7°**, the URL above says **14.7°**

The yellow cross (location marker) on NOAA images is in the correct place, so that the error cannot come from wrong coordinates.

Can you tell me, where the sun elevation is calculated?

**Reply by jp112sdl** [reply]
Okay, now it is clear for me: `scrtipts/tools/sun.py` uses the timezone from `~/.noaa-vs.conf` and therein I have to change it manually after switching between summer/standard time.
